api/sql/schema/370-rule.rate.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE [rule].[rate] (
+    rateId INT IDENTITY(1000, 1) NOT NULL,
+    conditionId INT NOT NULL,
+    targetCurrency VARCHAR(3) NOT NULL,
+    startAmount MONEY NOT NULL,
+    startAmountCurrency VARCHAR(3) NOT NULL,
+    startAmountDaily MONEY NOT NULL,
+    startCountDaily BIGINT NOT NULL,
+    startAmountWeekly MONEY NOT NULL,
+    startCountWeekly BIGINT NOT NULL,
+    startAmountMonthly MONEY NOT NULL,
+    startCountMonthly BIGINT NOT NULL,
+    rate FLOAT,
```

Comment:
float is an approximate-number data type. Long time ago I worked in a project with such columns and there were many problems caused by this type, use decimal/numeric